integrations/client/clientset/versioned/typed/flux.weave.works/v1beta1/fake/fake_helmrelease.go

```diff
@@ -97,6 +97,18 @@ func (c *FakeHelmReleases) Update(helmRelease *v1beta1.HelmRelease) (result *v1b
 	return obj.(*v1beta1.HelmRelease), err
 }
 
+// UpdateStatus was generated because the type contains a Status member.
+// Add a +genclient:noStatus comment above the type to avoid generating UpdateStatus().
```

Comment:
Wouldn't it be cleaner to place part of this comment at `v1beta1/types.go`?

e.g.
```
// Uncomment  to to avoid generating UpdateStatus()
// // +genclient:noStatus
```